PersonalBlog/urls.py

```diff
@@ -2,10 +2,18 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path,include
+from rest_framework import routers
+from BlogApp.views import UserViewSet, BlogPostViewSet
```

Comment:
Also Need to remove these codes from Project/urls.py
I have found better way to implement API. I think it would be better if we add path('api/', include(router.urls)), this code to the BlogApp/urls.py. With that way we can access to the BlogPosts api with this url: domain.com/apiBlogPosts
With the way that you have wrote we can access all of the urls with and without api/ . For example If we try enter domain.com/api -> this will redirect me to the home page of my BlogApp and this url domain.com/api/notifications/   will redirect me to the notifications page. Also I will be able to access that pages without the api/ . For example domain.com/notifications will be the same page with domain.com/api/notifications and this is not acceptable